_i18n/en.yml

```diff
@@ -0,0 +1,60 @@
+main-headline: Cargobike Routing
```

Comment:
we should have a consistent way of writing "cargo bike". I think its two words in english (and without "-")